debian/ocf-atool.postinst

```diff
@@ -0,0 +1,18 @@
+# create conf directory
```

Comment:
You can rename this file to simply `debian/postinst` since it is the only/first binary package. Also, put `#DEBHELPER#` on a line by itself to include auto-generated commands. Usually you can put that at the end of the file, but put it first in this file so `/etc/ocf-atool` directory will be created.